app/assets/stylesheets/dashboard.scss

```diff
@@ -0,0 +1,9 @@
+.dashboard_partial{
+  border: 2px solid #003A5B;
```

Comment:
Is there anything significant about that color? (also personally I like lowercase better in hex values)